ui/package-lock.json

```diff
@@ -1726,7 +1726,7 @@
     },
     "babel-register": {
       "version": "6.26.0",
-      "resolved": "https://registry.npmjs.org/babel-register/-/babel-register-6.26.0.tgz",
+      "resolved": "https://artifacts.netflix.com/api/npm/npm-netflix/babel-register/-/babel-register-6.26.0.tgz",
```

Comment:
Is this artifact available publicly outside of Netlix?

Reply:
Good catch. It should point to `registry.npmjs.org`. I'll revert locks update.